Extension/src/LanguageServer/configurations.ts

```diff
@@ -1498,15 +1518,15 @@ export class CppProperties {
         }
 
         // Validate paths (directories)
-        errors.includePath = this.validatePath(config.includePath);
+        errors.includePath = this.validatePath(config.includePath, {globPaths: true});
         errors.macFrameworkPath = this.validatePath(config.macFrameworkPath);
         errors.browsePath = this.validatePath(config.browse ? config.browse.path : undefined);
 
         // Validate files
-        errors.forcedInclude = this.validatePath(config.forcedInclude, false, true);
-        errors.compileCommands = this.validatePath(config.compileCommands, false);
-        errors.dotConfig = this.validatePath(config.dotConfig, false);
-        errors.databaseFilename = this.validatePath((config.browse ? config.browse.databaseFilename : undefined), false);
+        errors.forcedInclude = this.validatePath(config.forcedInclude, {isDirectory: false, skipRelativePaths: true, globPaths: true});
```

Comment:
I don't think forcedInclude should use the globPaths (forced includes are a specific file that gets inlined) -- can you remove that?

Reply:
Sure, Done.